install.rst

```diff
@@ -32,7 +32,9 @@ for scientific computing and data science.
   Anaconda::
 
     conda update conda
-    conda update ipython
+    conda update ipython ipython-notebook ipython-qtconsole
+
+  The last two entries are used to ensure that all the dependencies for the IPython notebook are installed and updated.  If the IPython notebook is not desired, the last two entries can be left off.
```

Comment:
While it should be clear from the package names, this suggests that `ipython-qtconsole` is required for the notebook. Perhaps, rather than putting packages in the example that can be left off, note that `ipython-notebook` can be installed for the notebook dependencies, `ipython-qtonconsole` for the qtconsole, etc.